.stylelintrc.json

```diff
@@ -1,11 +1,12 @@
 {
-	"extends": "stylelint-config-wordpress",
+	"extends": "@wordpress/stylelint-config",
 	"rules": {
 		"at-rule-empty-line-before": null,
 		"at-rule-no-unknown": null,
 		"comment-empty-line-before": null,
 		"declaration-block-no-duplicate-properties": null,
 		"declaration-property-unit-whitelist": null,
+		"declaration-property-unit-allowed-list": null,
```

Comment:
This rule produced errors. I tried to convert to warning unsuccessfully. We'll need to handle this in a follow-up. Actually the follow-up should re-examine all of our exceptions here and see why we have them and possibly setup prettier to fix our style files. Currently I can't turn on the automattic formatter for prettier for stylesheets in my IDE.